t/sdam_spec.t

```diff
@@ -127,6 +132,10 @@ sub check_outcome {
     is($topology->replica_set_name, $expected_set_name, 'correct setName for topology');
     is($topology->type, $outcome->{'topologyType'}, 'correct topology type');
     is($topology->logical_session_timeout_minutes, $outcome->{'logicalSessionTimeoutMinutes'}, 'correct ls timeout');
+    if ( defined $outcome->{'compatible'} ) {
```

Comment:
Here, if it's the `/too_old/` test file then we want to force the compatibility expectation to true. (Along with a comment that Perl has to support older servers.)

Reply:
Right, have done so :)